pages.de/common/cp.md

```diff
@@ -22,3 +22,7 @@
 - Kopiere alle Textdateien in einem Verzeichnis und warte auf eine Bestätigung, falls eine Datei überschrieben werden soll:
 
 `cp -i {{/pfad/zu/*.txt}} {{pfad/zu/ziel_verzeichnis}}`
+
+- Dereferenziere symbolische Links vor dem Kopieren:
```

Comment:
`Dereferenziere` sounds slightly weird. Perhaps there is a better term?

Reply:
I would say it sounds equally weird as `dereference` in English.
How about: `Folge symbolischen Links vor dem Kopieren:`?

Reply:
Sounds good to me. "Dereference" doesn't sound as weird in English in my experience since "reference" is a more common word in English than "referenzieren" in German… Perhaps we might also consider changing it in the English translation, however? "Follow symbolic links" is probably more clear to new users than "dereference symbolic links", at least it would be for me.

Reply:
Should I open a PR for what I suggested here?
The other conversations can be closed, I guess.

Reply:
Sounds good. Even `cp --help` uses `follow` instead of `dereference`.